auto_dev/test.py

```diff
@@ -62,5 +63,17 @@ def test_path(
     args = [path, *extra_args]
     os.environ["PYTHONPATH"] = "."
     os.environ["PYTHONWARNINGS"] = "ignore"
-    result = pytest.main(args)
-    return result == 0
+    os.environ["PYTHONPYCACHEPREFIX"] = "/dev/null"
+    os.environ["PYTHONDONTWRITEBYTECODE"] = "1"
+    import pytest
+
+    sys.dont_write_bytecode = True
+    result = pytest.main(args, )
+
+
+    # We remove the __pycache__ directories to avoid conflicts
+    result = (result == 0 or result == pytest.ExitCode.NO_TESTS_COLLECTED)
```

Comment:
This means we dont *need* tests in a component for it to pass the tests.

I am in a 2 minds about this;

A) it definitely does make it easier,

But the thing about tasking the path of least resistance is that we *SHOULD* have tests for everything we generate.